### ISSUE TITLE
Precise assembler in normal mode 1 ticking everything and not using power at MAX voltage or above

### ISSUE THREAD

**Issue by GDCloudstrike** [GTNewHorizons/GT-New-Horizons-Modpack#11320]
### Your GTNH Discord Username

GDCloud#2907

### Your Pack Version

2.2.0.0-RC-2

### Your Server

SP

### Your Expectation

Trying to OC the precise assembler to MAX voltage, expect it to OC normally.

### The Reality

The precise assembler in normal mode uses no power and does every recipe in 1 tick if the total power it can use goes above 1A MAX. Both tt hatches and normal hatches cause this to happen.

![image](https://user-images.githubusercontent.com/93287602/190507266-fc5574e2-2ddf-4764-91bd-17c3ac5d1103.png)

### Your Proposal

Fix the OC of the precise assembler to work properly for MAX voltage and above.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [ ] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by GlodBlock** [comment]
most gt's multi don't allow to overclock after max, otherwise it needs a special handler. so it will limit its max voltage to max after the fix.

**Comment by GDCloudstrike** [comment]
I downloaded the newest jenkins jar from 10 min ago and it appears that the issue still persists.

![image](https://user-images.githubusercontent.com/93287602/190700543-04deabcc-1ca1-4368-b9be-b8c0fba0311e.png)

**Comment by GlodBlock** [comment]
![image](https://user-images.githubusercontent.com/60341015/190860138-c5c62196-74cb-420f-bc13-557ba5cefafd.png)
doesn't it consume the energy from the hatch?

**Comment by GDCloudstrike** [comment]
No, it doesn't. That small consumption from the previous screenshot was me using wireless hatches (which dont continuously fill the entire energy buffer) and I ran a test in precise mode beforehand. In precise mode, it works as intended and also consumes power. 

In this screenshot I used normal, unpowered energy hatches and the machine still runs.
![image](https://user-images.githubusercontent.com/93287602/190863126-55d3f7f3-0693-4c7c-ad19-a9094ab342c3.png)

![2022-09-17_16 45 04](https://user-images.githubusercontent.com/93287602/190862719-a86727f4-5b5d-4560-aa23-418dfb12a87d.png)
